Shetland_PCPIP_Indicators.Rmd

```diff
@@ -588,3 +589,27 @@ pharma1_180_plotly
 pharma1_180_dt
 ```
 
+```{r export-all-data-frames, message=FALSE, warning=FALSE}
+# Create the named list of data frames and their sheet names
+all_data_frames <- list(
+  all_data = all_data,
+  monthly_event_type = monthly_event_type,
+  monthly_summary = monthly_summary,
```

Comment:
This one, and maybe other dataframes, contain multiple indicators.


Maybe something like.
```r
cv_invites <- monthly_summary |>
  select(PracticeID, census_date, cv_prev, cv_invites)

cv_attend <- monthly_summary |>
  select(PracticeID, census_date, cv_prev, cv_attends)
```

This code will go above the `all_data_frames <- list(` bit, so that they can then be included as separate sheets.